sql/moz-fx-data-shared-prod/addon_moderations_derived/cinder_decisions_raw_v1/query.py

```diff
@@ -129,14 +130,12 @@ def main():
 
     cinder_data = []
 
-    json_file = cinder_addon_decisions_download(date, bearer_token)
+    query_export = cinder_addon_decisions_download(date, bearer_token)
     """Data returns as a dictionary with a key called 'items' and the value being a list of data"""
-    query_export = check_json(json_file.text)
-    """Add date to each element in query_export for partitioning"""
     query_export_contents = query_export["items"]
+    """Add date to each element in query_export for partitioning"""
```

Comment:
The convention in Python is to use docstring comments only at the top of files/modules and the top of class/function definitions.  All other comments are typically just normal Python comments starting with `#`.